sexp.c

```diff
@@ -475,8 +475,11 @@ void sexp_init_context_globals (sexp ctx) {
   sexp_push(ctx, sexp_global(ctx, SEXP_G_FEATURES), SEXP_FALSE);
   sexp_car(sexp_global(ctx, SEXP_G_FEATURES)) = sexp_intern(ctx, (*(unsigned char*) &endianess_check) ? "little-endian" : "big-endian", -1);
   for (features=sexp_initial_features; *features; features++) {
-    sexp_push(ctx, sexp_global(ctx, SEXP_G_FEATURES), SEXP_FALSE);
-    sexp_car(sexp_global(ctx, SEXP_G_FEATURES)) = sexp_intern(ctx, *features, -1);
+    feature = sexp_intern(ctx, *features, -1);
+    if (sexp_not(sexp_memq(ctx, feature, sexp_global(ctx, SEXP_G_FEATURES)))) {
+      sexp_push(ctx, sexp_global(ctx, SEXP_G_FEATURES), SEXP_FALSE);
+      sexp_car(sexp_global(ctx, SEXP_G_FEATURES)) = feature;
```

Comment:
Either make feature a gc var, or call sexp_intern again here.

Note currently we never gc symbols so this isn't strictly needed, but that may change.

Reply:
I just pushed another commit to (hopefully) take care of this.  Please let me know if this isn't the correct way to go about fixing this problem.  Thanks!

Reply:
lgtm, thanks!